app/controllers/bulk_taggings_controller.rb

```diff
@@ -0,0 +1,37 @@
+class BulkTaggingsController < ApplicationController
+  def new
+    render :new, locals: { results: [] }
+  end
+
+  def search
+    if search_response.successful?
```

Comment:
What kind of errors do we expect here? Perhaps we should let the error bubble up.